### PR TITLE
Ticket2239: Make dataweb look like GUI

### PR DESCRIPTION
**Pull request by drapj002**
see https://github.com/ISISComputingGroup/IBEX/issues/2239

**To Test**
- Turn on or off instruments (e.g. demo) and see if the status and colour changes
- Connect with different users and see if users changes
- See if Inst. Time and Run Time change over time as expected

### CODE REVIEW

File: default.html [hunk 1]
```diff
@@ -7,12 +7,15 @@
...
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+        <script src="http://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js" type="text/javascript"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
         <script src="display_blocks.js" type="text/javascript" defer="defer"></script>
     </head>
 
     <body>
-        <div id="inst_name"></div>
+		<div id = "top_bar"></div>
+        <!-- <h3 style="padding: 10px; color: red; background-color: blue; border: black 2px solid" class = "text-center" id="inst_name"></h3></div> -->
```
Comment (Tom-Willemsen) [inline default.html]: remove commented code

File: display_blocks.js [hunk 2]
```diff
@@ -125,6 +125,7 @@ function refresh() {
 			displayError();
 		},
 		success: function(data){ 
+			//window.alert(JSON.stringify(data))
```
Comment (Tom-Willemsen) [inline display_blocks.js]: remove commented code

File: display_blocks.js [hunk 3]
```diff
@@ -161,10 +157,116 @@ function parseObject(obj) {
     var nodeInstPVList = document.createElement("ul");
 
     nodeInstPVs.appendChild(nodeInstPVList);
+    nodeInstPVs.appendChild(nodeInstPVList);
```
Comment (Tom-Willemsen) [inline display_blocks.js]: Why is this line needed twice? Please add comment to explain why it's needed (or remove if not needed).

File: display_blocks.js [hunk 4]
```diff
@@ -161,10 +157,116 @@ function parseObject(obj) {
...
+};
+
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+};
+
+function newPartOfTable(){
+	document.getElementById("next_part").removeAttribute("id");
+	document.getElementById("table_part").innerHTML = document.getElementById("table_part").innerHTML + "<th id = \"next_part\" style = \"padding: 10px; background-color:lightgrey ; border: black 2px solid\";></th>";
```
Comment (Tom-Willemsen) [inline display_blocks.js]: Use `+=`

File: display_blocks.js [hunk 5]
```diff
@@ -161,10 +157,116 @@ function parseObject(obj) {
...
+
+	var runTime = document.createElement("h5");
+	insidePart ="Run Time: "+obj["inst_pvs"]["RUNDURATION_PD"]["value"];
+	runTime.innerHTML = insidePart + "\xa0".repeat(30);
+	document.getElementById("next_part").appendChild(runTime);
+	
+	var currentTotalPeriods = document.createElement("h5");
+	insidePart ="Period: "+obj["inst_pvs"]["PERIOD"]["value"]+"/"+obj["inst_pvs"]["NUMPERIODS"]["value"];
+	currentTotalPeriods.innerHTML = insidePart + "\xa0".repeat(30);
+	document.getElementById("next_part").appendChild(currentTotalPeriods);
```
Comment (Tom-Willemsen) [inline display_blocks.js]: A lot of these bits look pretty close to identical. See if they can be refactored to use a common method.

File: display_blocks.js [hunk 6]
```diff
@@ -173,11 +275,12 @@ function displayError() {
 
     clear(nodeInstTitle);
     clear(nodeConfigTitle);
-
-    nodeInstTitle.appendChild(document.createTextNode(instrument));
+	//var instName = document.innerHTML(instrument)
+	//instName.class = "text-center"
+    //nodeInstTitle.appendChild(instName);
```
Comment (Tom-Willemsen) [inline display_blocks.js]: Remove commented code.

File: display_blocks.js [hunk 7]
```diff
@@ -161,10 +157,116 @@ function parseObject(obj) {
...
+
+function newPartOfTable(){
+	document.getElementById("next_part").removeAttribute("id");
+	document.getElementById("table_part").innerHTML = document.getElementById("table_part").innerHTML + "<th id = \"next_part\" style = \"padding: 10px; background-color:lightgrey ; border: black 2px solid\";></th>";
+};
+
+/**
+ * creates a Title at the top looking similar to the IBEX GUI
+ */
+function createTitle(obj){
```
Comment (Tom-Willemsen) [inline display_blocks.js]: Parameter should be named something else. This function does not work on any arbitrary object `obj`

File: display_blocks.js [hunk 8]
```diff
@@ -173,11 +275,12 @@ function displayError() {
...
     clear(nodeConfigTitle);
-
-    nodeInstTitle.appendChild(document.createTextNode(instrument));
+	//var instName = document.innerHTML(instrument)
+	//instName.class = "text-center"
+    //nodeInstTitle.appendChild(instName);
     nodeConfigTitle.appendChild(document.createTextNode("Could not connect to " + instrument + ", check IBEX server is running."));
 
-	document.getElementById("inst_name").appendChild(nodeInstTitle);
+	document.getElementById("top_bar").innerHTML = instrument;
```
Comment (Tom-Willemsen) [inline display_blocks.js]: I think this is meant to display an IBEX-style "FAKE is UNKNOWN" with a yellow background, but it doesn't seem to be working like that currently:

image

File: display_blocks.js [hunk 9]
```diff
@@ -160,24 +155,106 @@ function parseObject(obj) {
...
+    document.getElementById(elementID).innerHTML = "";
+}
+
+/**
+ * creates a Title at the top looking similar to the IBEX GUI
+ */
+function createTitle(inst_details){
+	clearBox("top_bar");
+	
+	document.getElementById("top_bar").innerHTML = "<div id = \"inst_name\"></div><table><tr id = table_part><th id = \"next_part\" style = \"padding: 10px; background-color:lightgrey ; border: black 2px solid\";></th></tr></table>";
```
Comment (Mukz95) [inline display_blocks.js]: addition of a style tag to make the width responsive to the page as the only the status of the top bar stretches across the page
